Container/Dockerfile.deeploy

```diff
@@ -21,17 +21,19 @@ COPY Makefile ./
 COPY requirements-dev.txt ./
 
 # Compile emulators
-RUN apt-get upgrade && \
-    apt-get update && \
-    apt-get install -y python3.10-venv \
-    python3.10-distutils 
-
 RUN --mount=type=cache,target=/ccache \
     ccache -z && \
     make pulp-sdk chimera-sdk qemu mempool banshee xtensor && \
     make gvsoc && \
     cp -r /app/toolchain/gvsoc/core/requirements.txt /app/core-requirements.txt && \
     cp -r /app/toolchain/gvsoc/gapy/requirements.txt /app/gapy-requirements.txt && \
+    ccache -s
+
+# Install SoftHier emulator and toolchain
+# WIESEP: We need to already clean up some space, otherwise the GitHub runners run out of disk space
+RUN --mount=type=cache,target=/ccache \
+    ccache -z && \
+    rm -rf toolchain/pulp-sdk toolchain/qemu toolchain/mempool toolchain/banshee toolchain/xtensor toolchain/xtl toolchain/xsimd toolchain/gvsoc && \
```

Comment:
I feel like this step makes more sense with the previous RUN command (we build and immediately clean up space there)